src/kbmod/run_search.py

```diff
@@ -329,8 +330,8 @@ def _count_known_matches(self, keep, img_info, search):
 
         # Extract a list of predicted positions for the final results.
         found_objects = []
-        for index in keep["final_results"]:
-            ppos = search.get_mult_traj_pos(keep["results"][index])
+        for row in result_list:
```

Comment:
the interface pretty much works the same as it did before so my koffi version of `_count_known_matches` won't really be affected 🙂